connaisseur/overlays/argocd/connaisseur-application.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
This file should be placed in https://github.com/operate-first/apps/tree/master/argocd/overlays/moc-infra/applications/envs/moc/smaug/operate-first